def get_clusters():
    'Synced API call to get all cluster names'
    instance = tornado.ioloop.IOLoop.instance()
    try:
        return instance.run_sync((lambda : API.get_clusters()))
    except Exception as ex:
        Log.error(('Error: %s' % str(ex)))
        Log.error('Failed to retrive clusters')
        raise
